NOTICE

```diff
@@ -1,13 +1,19 @@
 
    NGINX Unit.
 
-   Copyright 2017-2023 NGINX, Inc.
-   Copyright 2017-2023 Andrei Zeliankou
-   Copyright 2018-2023 Konstantin Pavlov
-   Copyright 2021-2023 Zhidao Hong
+   Copyright 2017-2024 NGINX, Inc.
+   Copyright 2017-2024 Andrei Zeliankou
+   Copyright 2018-2024 Konstantin Pavlov
+   Copyright 2021-2024 Zhidao Hong
+   Copyright 2022-2024 Andrew Clayton
+   Copyright 2022-2024 Liam Crilly
+   Copyright 2023-2024 Dan Callahan
+   Copyright 2023-2024 Danielle De Leo
+   Copyright 2023-2024 Dylan Arbour
+   Copyright 2023-2024 Gabor Javorszky
+   Copyright 2023-2024 Ippolitov Igor
```

Comment:
Should `Ippolitov Igor` be `Igor Ippolitov` here and elsewhere?

Also, while we're at it, can we change `Nginx, Inc.` to `NGINX, Inc.` in `src/test/nxt_rbtree1.c`, `src/test/nxt_rbtree1.h`, and `auto/endian`

Reply:
Thanks, fixed!